src/linear_algebra/matrix.lean

```diff
@@ -315,6 +315,21 @@ end
 
 end vector_space
 
+section finite_dimensional
+
+variables {R : Type v} [field R]
+
+instance finite_dimensional_matrix : finite_dimensional R (matrix m n R) :=
```

Comment:
You are already in the `matrix` namespace, so lean will autogenerate the name `matrix.finite_dimensional` for this instance.
```suggestion
instance : finite_dimensional R (matrix m n R) :=
```